test/integration/features/cmd-image.feature

```diff
@@ -1,4 +1,4 @@
-@cmd-image @command
+@cmd-image @command @minishift-only
```

Comment:
@amitkrout I am not sure why is this should be only minishift specific test?

Reply:
@LalatenduMohanty My understanding here is the major scenarios are not for downstream i.e. image exporting and importing which is the key feature of image caching. There is no point keeping the rest scenarios without this and it is very difficult to split such major scenario for tagging. So i tagged whole file minishift specific.

This file will be skipped if i run this file with downstream binary which is a plus IMO rather throwing error. And its good enough to modify the same file for downstream use.

Reply:
@amitkrout @LalatenduMohanty +1 for modifying it for downstream use. Mostly below scenario
```
     Then stdout should match "Importing 'openshift\/origin:v[0-9]+\.[0-9]+\.[0-9]+' [\.]+ OK"
      And stdout should match "Importing 'openshift\/origin-docker-registry:v[0-9]+\.[0-9]+\.[0-9]+' [\.]+ OK"
      And stdout should match "Importing 'openshift\/origin-haproxy-router:v[0-9]+\.[0-9]+\.[0-9]+' [\.]+ OK"
```

Reply:
@amitkrout I am not favor of marking this with minishift-only tag. In downstream we should modify the tests and if the rebase causes more issues, we can mark explicit tests (which fails for downstream) and add cdk-only scenarios in future.